docs/GET_STARTED.md

```diff
@@ -206,3 +206,32 @@ Hinweis: KoliBri-Inputs übergeben in der Regel das Ursprungsevent als ersten Pa
 
 
 
+
+### III ohne Framework
+
+Hinweis: ohne einen Bundler ist KoliBri aktuell nicht verwendbar.
+
+`pnpm i @public-ui/core @public-ui/components @public-ui/themes`
```

Comment:
Kannst Du bitte überall `@public-ui/core` entfernen. Es steckt alles in `@public-ui/components`